README.md

```diff
@@ -33,177 +35,3 @@ We love and welcome contributions to our front-end repository which can be found
 
 <!-- ALL-CONTRIBUTORS-LIST:END -->
 
-## Prerequisites
-
--   [Node](https://nodejs.org)
-    -   We recommend using [nvm](https://github.com/nvm-sh/nvm) and running `nvm use` in this directory to sync with the project's Node version. However, if you wish to install Node your own way and ensure a consistent version with `.nvmrc`, that's fine too
--   [Yarn Berry](https://yarnpkg.com/getting-started/install)
--   [Docker Desktop](https://www.docker.com/products/docker-desktop/)
-
-# Setup
```

Comment:
None of this setup information is found in the links to Gitbook. I'd suggestion not removing any of it until all of the instructions have been moved to the documentation.

Reply:
This comment was resolved on [Slack](https://find-a-doc.slack.com/archives/C06QP1QLX2N/p1738828538929809).